.gitlab-ci.yml

```
@@ -1,244 +1,10 @@
 stages:
   - test
-  - publish
-  - trigger_prep
-  - trigger
-
-variables:
-  LICENSE_HEADERS_IGNORE_FILES_REGEXP: "./ltmain.sh"
-  DEBIAN_FRONTEND: noninteractive
-
-include:
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-github-status-updates.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-commits.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-license.yml'
-  - project: 'Northern.tech/Mender/mendertesting'
-    file: '.gitlab-ci-check-c++.yml'
-
-variables:
-  GIT_SUBMODULE_STRATEGY: recursive
-
-test:no-tests:
-  stage: test
-  extends: .test:static-template
-  script:
-    - cmake -D BUILD_TESTS=OFF .
-    - make --jobs=$(nproc --all) --keep-going
-  tags:
-    - mender-qa-worker-generic
 
 test:unit:
   stage: test
   image: debian:11
```

Nitpick: Use busybox for security hardening.
```suggestion
  image: busybox:stable-musl
```